mail-client/thunderbird-bin/thunderbird-bin-68.0.ebuild

```diff
@@ -0,0 +1,182 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+MOZ_ESR=""
+MOZ_LIGHTNING_VER="68.0b6"
+
+# Can be updated using scripts/get_langs.sh from mozilla overlay
+MOZ_LANGS=(ar ast be bg br ca cs cy da de el en en-GB en-US es-AR
+es-ES et eu fi fr fy-NL ga-IE gd gl he hr hsb hu hy-AM id is it ja ko lt
+nb-NO nl nn-NO pl pt-BR pt-PT rm ro ru si sk sl sq sr sv-SE tr
+uk vi zh-CN zh-TW )
+
+# Convert the ebuild version to the upstream mozilla version, used by
+MOZ_PN="${PN/-bin}"
+MOZ_PV="${PV/_beta/b}"
+MOZ_PV="${MOZ_PV/_rc/rc}"
+
+if [[ ${MOZ_ESR} == 1 ]]; then
+	# ESR releases have slightly version numbers
+	MOZ_PV="${MOZ_PV}esr"
+fi
+
+MOZ_P="${MOZ_PN}-${MOZ_PV}"
+
+MOZ_HTTP_URI="https://archive.mozilla.org/pub/${MOZ_PN}/releases"
+
+inherit eutils multilib pax-utils xdg-utils nsplugins mozlinguas-v2
+
+DESCRIPTION="Thunderbird Mail Client"
+SRC_URI="${SRC_URI}
+	amd64? ( ${MOZ_HTTP_URI}/${MOZ_PV}/linux-x86_64/en-US/${MOZ_P}.tar.bz2 -> ${PN}_x86_64-${PV}.tar.bz2 )
+	x86? ( ${MOZ_HTTP_URI}/${MOZ_PV}/linux-i686/en-US/${MOZ_P}.tar.bz2 -> ${PN}_i686-${PV}.tar.bz2 )
+	https://dev.gentoo.org/~juippis/distfiles/lightning-${MOZ_LIGHTNING_VER}.tar.xz"
+# the below only works when upstream releases the xpi with all locales bundled
+#	${MOZ_HTTP_URI/${MOZ_PN}/calendar/lightning}/${MOZ_LIGHTNING_VER}/linux/lightning.xpi -> lightning-${MOZ_LIGHTNING_VER}.xpi
+
+HOMEPAGE="https://www.thunderbird.net/"
+RESTRICT="strip mirror"
+
+KEYWORDS="-* ~amd64 ~x86"
+SLOT="0"
+LICENSE="MPL-2.0 GPL-2 LGPL-2.1"
+IUSE="+crashreporter +ffmpeg +pulseaudio selinux"
+
+DEPEND="app-arch/unzip
+	app-arch/zip"
+
+RDEPEND="dev-libs/atk
+	>=sys-apps/dbus-0.60
+	>=dev-libs/dbus-glib-0.72
+	>=dev-libs/glib-2.26:2
+	>=media-libs/alsa-lib-1.0.16
+	media-libs/fontconfig
+	>=media-libs/freetype-2.4.10:2
+	virtual/freedesktop-icon-theme
+	>=x11-libs/cairo-1.10[X]
+	x11-libs/gdk-pixbuf:2
+	>=x11-libs/gtk+-2.18:2
+	>=x11-libs/gtk+-3.4.0:3
+	x11-libs/libX11
+	x11-libs/libXext
+	x11-libs/libXrender
+	x11-libs/libXt
+	>=x11-libs/pango-1.22.0
+	pulseaudio? ( !<media-sound/apulse-0.1.9
+		|| ( media-sound/pulseaudio media-sound/apulse ) )
+	ffmpeg? ( media-video/ffmpeg )
+	crashreporter? ( net-misc/curl )
+	selinux? ( sec-policy/selinux-thunderbird )
+"
+
+QA_PREBUILT="
+	opt/${MOZ_PN}/*.so
+	opt/${MOZ_PN}/${MOZ_PN}
+	opt/${MOZ_PN}/${PN}
+	opt/${MOZ_PN}/crashreporter
+	opt/${MOZ_PN}/pingsender
+	opt/${MOZ_PN}/plugin-container
+	opt/${MOZ_PN}/minidump-analyzer
+	opt/${MOZ_PN}/mozilla-xremote-client
+	opt/${MOZ_PN}/updater
+"
+
+S="${WORKDIR}/${MOZ_PN}"
+
+src_unpack() {
+	unpack ${A}
+
+	# Unpack language packs
+	mozlinguas_src_unpack
+	#xpi_unpack lightning-${MOZ_LIGHTNING_VER}.xpi
```

Comment:
Delete this?

Reply:
It's not my ebuild so I'm afraid taking off these maintainer notes, if they are needed some day (not THAT familiar with this...) :I 
This ebuild does have some other parts I'd also clean...